src/Settings/Features.php

```diff
@@ -113,7 +116,10 @@ public static function all_features(): array {
 			NewDesignCountrySelector::class,
 			NewDesignNavigationBar::class,
 			GoogleSheetReplacesSmartsheet::class,
-			ListingPagePagination::class,
+			TagPagePagination::class,
```

Comment:
These new features show as `Dev only`, is that intended? Because from the code it seems they shouldn't 🤔 

<img width="620" alt="Screenshot 2022-05-03 at 09 34 04" src="https://user-images.githubusercontent.com/6949075/166418327-f198bdf1-1256-4e37-84ba-1a7a8b0f084d.png">

Reply:
You mean the file location, not being in `/Dev/` namespace? If that, it's intended, creating that namespace was a mistake as we would need to rename stuff too much, it's nicer to have it grouped logically.

Reply:
No, I meant that it's not located under the `// Dev only` comment a few lines below

Reply:
It's a similar issue, it's not nice to maintain when a feature goes from dev to prod. Like the PR description mentions I'm still checking the improve the exact location and for these features.